RefugeeLand.Core.Api/Brokers/Storages/IStorageBroker.Shelters.cs

```diff
@@ -11,5 +11,6 @@ namespace RefugeeLand.Core.Api.Brokers.Storages
     public partial interface IStorageBroker
     {
         ValueTask<Shelter> InsertShelterAsync(Shelter shelter);
+        IQueryable<Shelter> SelectAllShelters();
```

Comment:
@TylerDDev You are missing a using for LinQ on top of the file in order to use the IQueryable type.